frontend/src/views/SqlEditor/EditorPanel/QueryEditor.vue

```diff
@@ -47,6 +48,17 @@ const selectedInstanceEngine = computed(() => {
   ) as string;
 });
 
+const selectedLanguage = computed(() => {
```

Comment:
We now have a uppercase from backend API and formated engine name looks like `MySQL`/`PostgreSQL` in frontend which is more standard. So I think we can use the formated name instead of the lowercase.
https://github.com/bytebase/bytebase/blob/main/frontend/src/store/modules/instance.ts#L217

Reply:
> We now have a uppercase from backend API and formated engine name looks like `MySQL`/`PostgreSQL` in frontend which is more standard. So I think we can use the formated name instead of the lowercase. https://github.com/bytebase/bytebase/blob/main/frontend/src/store/modules/instance.ts#L217

there is for the monaco-editor' language option, which support: `mysql`、`pgsql`、`sql`

Reply:
Fine.